### DISCUSSION TITLE
List of Components and Suggestions

### DISCUSSION THREAD

**Discussion by zainphp** [apa-developer/apa-ui#17] [Ideas]
> [!TIP]
> See the draft of Apa UI in [Figma](https://www.figma.com/design/7BXrgZAKwqwjkYXDvIfs75/APA-UI?node-id=0-1&t=mKnpnPrgqpa46iJq-1).

## Current available components:

- [⚙️ \<Button />](https://github.com/apa-developer/apa-ui/labels/Button)
- [⚙️ \<Checkbox />](https://github.com/apa-developer/apa-ui/labels/Checkbox)
- [⚙️ \<Dropdown />](https://github.com/apa-developer/apa-ui/labels/Dropdown)
- [⚙️ \<Input />](https://github.com/apa-developer/apa-ui/labels/Input)
- [⚙️ \<Radio />](https://github.com/apa-developer/apa-ui/labels/Radio )
- [⚙️ \<Select />](https://github.com/apa-developer/apa-ui/labels/Select)
- [⚙️ \<Textarea />](https://github.com/apa-developer/apa-ui/labels/Textarea)

### Suggestions:

- Alert
- Avatar
- Badge
- Card
- Dialog
- ❌ ~Layouting~
  - ❌ ~Container~
  - ❌ ~Grid~
  - ❌ ~Stack~
- Progress
- Separator/Divider
- Sidebar
- Skeleton
- Slider
- Switch
- Table
- Toast/Snackbar
- Tooltip

## Supported frameworks:

- [⚙️ React](https://github.com/apa-developer/apa-ui/labels/React)
- [⚙️ Svelte](https://github.com/apa-developer/apa-ui/labels/Svelte)
- 📝 Laravel's Blade

> [!NOTE]
> Icons Legend:
> | Icon | Description      |
> |:----:|------------------|
> |   ✅  | Available        |
> |   ⚙️  | Work in progress |
> |   📝  | Planned          |
> |   ❌  | Not Implemented  |

**Comment by powxenv** [comment]
i don't think we should do layouting stuff

**Reply by zainphp** [reply]
namanya juga saran heo
